config.md

```diff
@@ -101,6 +101,14 @@ For POSIX platforms the `mounts` structure has the following fields:
 * **`type`** (string, OPTIONAL) The type of the filesystem to be mounted.
     * Linux: filesystem types supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660"). For bind mounts (when `options` include either `bind` or `rbind`), the type is a dummy, often "none" (not listed in */proc/filesystems*).
     * Solaris: corresponds to "type" of the fs resource in [zonecfg(1M)][zonecfg.1m].
+* **`uidMappings`** (array of type LinuxIDMapping, OPTIONAL) The mapping to convert UIDs from the source file system to the destination mount point.
+This mapping contains the following fields: ContainerID, HostID, Size.
+The Size field is a range for mapping, as an example.
+        1000 1001 1       -> map UID 1000 to uid 1001 in <target> and no other UIDs
+        20000 100000 1000 -> map UID 20000 to uid 100000, uid 20001 to UID 100001 [...] until UID 20999 in <target>
```

Comment:
The formatting here renders a little bit odd (all squashed together into a single paragraph). :grimacing:

Reply:
Since this `LinuxIDMapping` isn't a new type, is there somewhere else this description should perhaps go?

Reply:
I fixed formatting, just adding new line.
Do you suggest to add reference to some man page or commit message of kernel's patch?

Reply:
> Since this LinuxIDMapping isn't a new type, is there somewhere else this description should perhaps go?

@AlexeyPerevalov I guess that @tianon means that the format description should also be added to the existing documentation that mentions/uses LinuxIDMapping. Specifically, this is https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md#user-namespace-mappings

Now I see that the way you describe it is the way it is expected by the kernel (three space-separated values). In here, we use three named fields, so space-separated format makes less sense.

I guess you can just ditch the example, instead referring to https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md#user-namespace-mappings